### DISCUSSION TITLE
1列(7キーの列)が反応しません。助けてください。

### DISCUSSION THREAD

**Discussion by hidazin** [Yowkees/keyball#275] [Q&A] [answered]
お世話になっております。
マウスの利用に首肩腕手首に疲れを感じていた頃キーボードにトラックボールがついているKeyballに驚き、しかも地元でキーボードを開発された方がいる!?!?!とさらに驚き、初めて自作キーボードを購入させていただきました。
さて本題ですが、7キーがある１列が丸っと反応しなくなりました。いっとき使えていたのですが再び反応しなくなりました。この場合どこを確認すればよいでしょうか？
【状況】
1. 組み立て直後に7キー列が反応しなかったため、PCBとピンソケット、およびPro Microとピンヘッダを再度はんだ付け
3. １週間くらい使えた
4. 本日再び7キー列反応せず
5. 左右のPro Microを入れ替えるが同じく7キー列が反応せず
6. 再度してはんだ付けするも反応せず

**Comment by Yowkees** [comment]
飛騨の方ですか！地元をはじめ各地いろんな方に興味を持っていただけて嬉しい限りです。

さて、１列すべて動かないキーがあるということで、半田付け不良で間違いないと思います。
確認してみますので、半田付け面のお写真を見せていただけますでしょうか？

お手数ですがご対応お願いいたします。

**Reply by hidazin** [reply]
ご連絡ありがとうございます。
写真を撮ってみました。これでわかりますでしょうか？
![image](https://github.com/Yowkees/keyball/assets/139552999/2f90beae-7a19-4d80-8cc7-49561383bf04)
![image](https://github.com/Yowkees/keyball/assets/139552999/2a1b0210-581f-44cd-9a32-0fdc3f848572)
![image](https://github.com/Yowkees/keyball/assets/139552999/64e10ff5-065d-4633-ac54-02724e6386f9)
![image](https://github.com/Yowkees/keyball/assets/139552999/1ba4cb50-d457-4fd5-b3ab-0c8d8ec540c8)

**Comment by Yowkees** [comment, marked as answer]
ご連絡が遅くなり申し訳ありません。
ProMicro裏の半田付けはうまくできていると思いますので、ダイオードの半田付けが怪しいかもしれません。
１列すべてのキーに影響するのは、下記画像の赤丸のダイオードですので、こちらの半田付けもご確認いただければ幸いです。
以上、お手数ですがご確認お願いいたします。

![KB61R_key_fix](https://github.com/Yowkees/keyball/assets/84209772/bde55298-18b3-4f72-a690-cca060acd32c)

**Reply by hidazin** [reply]
ご回答ありがとうございます。
まさにご指摘の通りダイオードの接触不良でした。
いったん外して確認しようと思い片方に半田ごてを当てたらポロッと外れましたwww
両端を再度はんだ付けして正常に動作するようになりました。
この度はありがとうございました。大変助かりました。

東京は連日猛暑でエアコンを消す時がないほどですが、古川も猛暑だろうと思います。どうぞご自愛ください。
昔はもっと涼しくて蛍も天の川も見えたんですがねぇ...

**Comment by Yowkees** [comment]
ご対応ありがとうございました。無事に直って安心しました！
ダイオードのテンプラは、半田ゴテで触ると部品が脱落するので分かりやすいですね。

飛騨も猛暑ですが、きっと東京のほうが大変な時期かと思います。ぜひご自愛くださいませ。

PS.私の家は山のほうでして、なんとか蛍も天の川も楽しんでおりま～す☆
